settings.gradle

```diff
@@ -1,22 +1,17 @@
 import com.gradle.scan.plugin.PublishedBuildScan
 
-
 pluginManagement {
     repositories {
+        // # Gradle looks for dependency artifacts in repositories listed in 'repositories' blocks in descending order.
+
+        // ## Prefer repos controlled by Airbyte.
+        // TODO: add airbyte-controlled proxy repos here
+
+        // ## Look into other, public repos.
```

Comment:
❓Does the markdown get registered in here? or should this be

```
// Look into other, public repos.
```

❗Also I think this could use some more context, I dont know what you mean

Reply:
The markdown is purely decorative to highlight an otherwise implicit hierarchy.

I'd like another reviewer familiar with gradle to weigh in on the understandability of the commentary. On the one hand, I know my prose often sucks, on the other hand, this might require familiarity with java. I honestly don't know.